Documentation/mobile-manual-v3.txt

```diff
@@ -144,15 +144,15 @@ The _Subsurface_ maintainer provides a dedicated and
 free Internet cloud storage account for users of _Subsurface_ and _Subsurface-mobile_.
 You can use this to store your dive log and to share your data between various computers
 and devices. The _Subsurface_ team never accesses your dive data without explicit
-permission to do so, the data are not used for any purpose other than providing them to the user who
+permission to do so, the data is not used for any other purpose than providing them to the user who
```

Comment:
I know, I know. People are doing the incorrect "data as singular" all the time.
But it bugs the hell out of me.
There is one datum that proves your point, but the data are clear... the word "data" is plural...